wallet/test/test_utils.h

```diff
@@ -21,4 +21,7 @@
 
 #define CHECK(cond) CHECK_MSG(cond,"test condition failed");
 
+#include "lightningd/lightningd.h"
```

Comment:
This include likely needs to be moved to the top.

Reply:
addressed af1c99e6d704b8ead29a9d802721b399f35c0838